### PR TITLE
Add default project option

### PR DESCRIPTION
**Pull request by 0x62ash**

### CODE REVIEW

File: options.html [hunk 1]
```diff
@@ -31,7 +31,16 @@
 				</label>
 			</td>
 			<td>
-				<input type='text' name='urlOption' id='urlOption' maxlength='255' size='30' autofocus />&nbsp;<code>/browse/KEY</code>
+				<input type='text' name='urlOption' id='urlOption' maxlength='255' size='30' autofocus />&nbsp;<code>/browse/PROJECT-123</code>
+			</td>
+		</tr>
+		<tr>
+			<td>
+				<label for='urlOption'><b>Default project:</b>
```
Comment (iriiiina) [inline options.html]: Please add description of this field to users: it's not mandatory and it works only when user inserts numbers.

File: options.html [hunk 2]
```diff
@@ -31,7 +31,16 @@
 				</label>
 			</td>
 			<td>
-				<input type='text' name='urlOption' id='urlOption' maxlength='255' size='30' autofocus />&nbsp;<code>/browse/KEY</code>
+				<input type='text' name='urlOption' id='urlOption' maxlength='255' size='30' autofocus />&nbsp;<code>/browse/PROJECT-123</code>
```
Comment (iriiiina) [inline options.html]: PROJECT is actually not always in use, you can add some description about cases when predefined project is in use.

File: options.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html>
 
 <head>
-    <title>Options of JIRA Issue Opener 2.0</title>
+    <title>Options of JIRA Issue Opener 2.1</title>
```
Comment (iriiiina) [inline options.html]: Version number in the header should be also changed: <h1>Options of JIRA Issue Opener 2.0</h1>

File: options.html [hunk 4]
```diff
@@ -49,4 +58,4 @@
 
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (iriiiina) [inline options.html]: Please also update README.md file: describe new functionality there.

**Comment by 0x62ash** [comment]
Done

**Pull request merged**